spec/lib/administrate/search_spec.rb

```diff
@@ -29,14 +29,21 @@ class MockDashboardWithAssociation
       searchable: true,
       searchable_field: "name",
     ),
+    author: Administrate::Field::BelongsTo.with_options(
+      searchable: true,
+      searchable_fields: ["first_name", "last_name"],
+    ),
     address: Administrate::Field::HasOne.with_options(
       searchable: true,
-      searchable_field: "street",
+      searchable_fields: ["street"],
     ),
   }.freeze
 end
 
 describe Administrate::Search do
+  before { ActiveSupport::Deprecation.silenced = true }
+  after { ActiveSupport::Deprecation.silenced = false }
```

Comment:
I think stubbing this with RSpec's facilities might be marginally better, as it automatically un-stubs after the example. What do you think?

```ruby
before do
  allow(ActiveSupport::Deprecation).to receive(:silenced).and_return(true)
end
```

Reply:
To be honest I don't like this approach, it feels semantically wrong to use stubs for this use case. I think current version shows intent quite well and version with stubs only provides marginal syntactical improvements. I can change it if you feel strongly about it though.

Reply:
Nah, it's ok. Thank you for your thoughts :+1: